wled00/fcn_declare.h

```diff
@@ -318,6 +318,46 @@ void setCountdown();
 byte weekdayMondayFirst();
 void checkTimers();
 void calculateSunriseAndSunset();
+
+// Timer management functions
+void addTimer(uint8_t preset, uint8_t hour, int8_t minute, uint8_t weekdays, uint8_t monthStart, uint8_t monthEnd, uint8_t dayStart, uint8_t dayEnd);
+void clearTimers();
+void syncTimersToArrays();
+uint8_t getTimerCount();
+uint8_t getRegularTimerCount();
+bool hasSunriseTimer();
+bool hasSunsetTimer();
+
+// Timer constants
+const uint8_t maxTimePresets = 32;
```

Comment:
Use `constexpr` or `#define` if this is pure constant so that compiler can optimize.